src/smart-components/group/group-table-helpers.js

```diff
@@ -40,44 +42,158 @@ DefaultPlatformPopover.propTypes = {
   bodyContent: PropTypes.string.isRequired,
 };
 
-export const createRows = (isAdmin, data, selectedRows = []) => {
+export const createRows = (isAdmin, data, selectedRows, expanded = [], isLoading) => {
   const intl = useIntl();
   return data.reduce(
-    (acc, { uuid, name, roleCount, principalCount, modified, platform_default: isPlatformDefault, admin_default: isAdminDefault }) => [
+    (
+      acc,
+      { uuid, name, roleCount, principalCount, modified, roles, members, platform_default: isPlatformDefault, admin_default: isAdminDefault },
+      i
+    ) => [
       ...acc,
       {
         uuid,
         isAdminDefault,
         isPlatformDefault,
         cells: [
-          <Fragment key={uuid}>
-            <div className="pf-m-inline-flex">
-              {isAdmin ? (
+          {
+            title: isAdmin ? (
+              <>
                 <AppLink
                   key={`${uuid}-link`}
                   state={{ uuid }}
                   to={pathnames['group-detail'].link.replace(':groupId', isPlatformDefault ? 'default-access' : uuid)}
                 >
                   {name}
                 </AppLink>
+                <Fragment>
+                  {(isPlatformDefault || isAdminDefault) && (
+                    <DefaultPlatformPopover
+                      id={`default${isAdminDefault ? '-admin' : ''}-group-popover`}
+                      uuid={uuid}
+                      key={`${uuid}-popover`}
+                      bodyContent={intl.formatMessage(isAdminDefault ? messages.orgAdminInheritedRoles : messages.usersInheritedRoles)}
+                    />
+                  )}
+                </Fragment>
+              </>
+            ) : (
+              name
+            ),
+          },
+          { title: roleCount, props: { isOpen: expanded[uuid] === 2 } },
+          {
+            title: principalCount,
```

Comment:
a padding seems to be missing
```suggestion
            <div className={classNames({ 'pf-u-p-lg': isPlatformDefault || isAdminDefault })}>{principalCount}</div>,
```